README.md

```diff
@@ -35,10 +34,17 @@ go get -u github.com/haya14busa/gopkgs/cmd/gopkgs
 ```
 $ gopkgs -h
 Usage of gopkgs:
-  -fullpath
```

Comment:
Please do not remove existing flags. It's not backward compatible. I understand we can use "-f" instead but I believe "-fullpath" and "-short" is handy.

Reply:
I put back the `-fullpath` and `-short` with additional `-format` for the custom one